src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,63 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { Tab } from '../../types/Tab';
+import { TabInfo } from '../TabInfo';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+function selectTab(tabs: Tab[], selectedTabId: string) {
+  return tabs.find((tab) => tab.id === selectedTabId) || tabs[0];
+}
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const [selectedTab, setSelectedTab] = useState(
+    selectTab(tabs, selectedTabId),
+  );
+  const { id, content } = selectedTab;
+
+  const handleTabClick = (tab: Tab) => {
+    if (tab.id !== id) {
+      setSelectedTab(tab);
+      onTabSelected(tab);
+    }
+  };
```

Comment:
you don't need state here
```suggestion
  const selectedTab = selectTab(tabs, selectedTabId);
  
  const { id, content } = selectedTab;

  const handleTabClick = (tab: Tab) => {
    if (tab.id !== id) {
      onTabSelected(tab);
    }
  };
```